solr/modules/sql/build.gradle

```diff
@@ -20,6 +20,7 @@ apply plugin: 'java-library'
 description = 'SQL Module'
 
 dependencies {
+  implementation platform(project(':platform'))
```

Comment:
The explicit-ness of this is good, I think, even if it means an additional line everywhere.  Maybe another different approach would be for the build to insert these automatically.  Either is fine.